awx_collection/plugins/modules/tower_host.py

```diff
@@ -89,7 +89,7 @@ def main():
     )
 
     # Create a module for ourselves
-    module = TowerModule(argument_spec=argument_spec)
+    module = TowerAPIModule(argument_spec=argument_spec)
```

Comment:
The new import/export code does not support hosts or groups, though there is an open issue to add them in.